threads/test_thread_rand.c

```diff
@@ -0,0 +1,88 @@
+#include <string.h>
```

Comment:
File should start with the properly constructed label as it is described in documentation.

Reply:
Yes, my mistake, label added.

Reply:
label needs fixing, please read what You're copying!